packages/kit-bg/src/vaults/impls/neo/sdkNeo/signMessage.ts

```
@@ -0,0 +1,108 @@
+/**
+ * Verifies a signature
+ * @param hex - Message to verify (hex format)
+ * @param sig - Signature (hex format)
+ * @param publicKey - Public key (hex format)
+ * @returns Verification result, true for valid signature
+ */
+export function verify(hex: string, sig: string, publicKey: string): boolean {
+  try {
+    if (!isPublicKey(publicKey, true)) {
+      // eslint-disable-next-line no-param-reassign
+      publicKey = getPublicKeyUnencoded(publicKey);
```

Biome (1.9.4)

[error] Reassigning a function parameter is confusing.

The parameter is declared here:

Use a local variable instead.

(lint/style/noParameterAssign)